### PR TITLE
Update Dependencies - Part 2

### PR DESCRIPTION
**Pull request by SyntaxNode**
**github.com/prometheus/client_golang**
**github.com/rcrowley/go-metrics**
**github.com/vrischmann/go-metrics-influxdb**
All metrics related dependencies have been updated to the same versions we are using in Prebid Server. We haven't seen any issues with these updates in Prebid Server, so I expect no issues for Prebid Cache. This does include breaking changes / new required fields for Influx metrics. Please see the details on the Prebid Server release notes: https://github.com/prebid/prebid-server/releases/tag/v0.198.0

**github.com/sirupsen/logrus**
We were 6 releases behind. The release notes indicate mostly just bug fixes.

**github.com/stretchr/testify**
We were only 1 minor release behind. This is used exclusively for tests and tests still pass.

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -253,23 +253,32 @@ const (
...
-	Username string `mapstructure:"username"`
-	Password string `mapstructure:"password"`
-	Enabled  bool   `mapstructure:"enabled"`
+	Enabled         bool   `mapstructure:"enabled"`
+	Host            string `mapstructure:"host"`
+	Database        string `mapstructure:"database"`
+	Measurement     string `mapstructure:"measurement"`
+	Username        string `mapstructure:"username"`
+	Password        string `mapstructure:"password"`
+	AlignTimestamps bool   `mapstructure:"align_timestamps"`
```
Comment (VeronikaSolovei9) [inline config/config.go:262]: Should these two new values have to be added to `func setConfigDefaults(v *viper.Viper) `?

File: metrics/influx/influx.go [hunk 2]
```diff
@@ -10,7 +10,7 @@ import (
 	influxdb "github.com/vrischmann/go-metrics-influxdb"
 )
 
-var TenSeconds time.Duration = time.Second * 10
+const TenSeconds time.Duration = time.Second * 10
```
Comment (VeronikaSolovei9) [inline metrics/influx/influx.go:13]: :+1:

**Review by VeronikaSolovei9 — COMMENTED** [review]
Local testing looks good

**Review by guscarreon — APPROVED** [review]
LGTM

**Pull request merged**